content/engine/install/raspberry-pi-os.md

```diff
@@ -53,9 +53,9 @@ To install Docker Engine, you need one of the following OS versions:
 
 Before you can install Docker Engine, you need to uninstall any conflicting packages.
 
-Distro maintainers provide an unofficial distributions of Docker packages in
-APT. You must uninstall these packages before you can install the official
-version of Docker Engine.
+Distro maintainers provide unofficial distributions of Docker packages in
```

Comment:
Why does CI invalidate something that was already there (and is used in the other variants of this page?

<img width="1503" alt="Screenshot 2024-07-23 at 17 28 12" src="https://github.com/user-attachments/assets/a9a2fbdf-0625-4c87-b07a-233836896b11">

Reply:
Reverted this change for now, but we should look how we can tweak these 🤔